regression-tests/25-compile-c64/Makefile

```diff
@@ -0,0 +1,14 @@
+EXAMPLESDIR=../../examples
+TOOLSDIR=../../tools
+
+# build c64 tests. Many examples won't fit in memory or won't compile due to the
+# lack of int64_t in cc65, so only a subset of IPv4 is tested.
+EXAMPLES = irc/c64 \
+telnet-server/c64 \
+webbrowser/c64 \
+webserver/c64 \
+wget/c64
```

Comment:
> Enclosing #if/#endif are missing there, I believe surrounding lines 1857/1858 would solve the problem without breaking anything else, could anyone confirm that?

The code section in question is conceptually the same for IPv4 and IPv6. So I checked out the IPv4 code (which builds fine without UIP_ACTIVE_OPEN) for additional insights. The whole nested "if"'s are missing in IPv4, see https://github.com/contiki-os/contiki/blob/f9a30236c87741259542210fc53340c39a7829ca/core/net/ipv4/uip.c#L1473. Git blame for IPv6 shows https://github.com/contiki-os/contiki/commit/f31d6f271fe7b1989fb335a251cfa07a22f83f57 meaning that Adam provided a fix to IPv6 (which might likely make sense for IPv4 too) very likely without taking UIP_ACTIVE_OPEN into account. I consider this another indication that the #if/#endif in question is indeed the correct approach.

> Linkage fails due to code size (overflows by 4849 bytes).

Ooops, that's a lot :-( And this for hello-world which basically does nothing? I asn't aware that IPv6 is soo much bigger than IPv4. I can tell you for sure that you won't find any cc65 compiler/linker/setup magic to improve the situation. As space is always very tight for the cc65 targets this is all already tuned to the limit. The only thing you can do is find Contiki (CFG_...) macros that reduce/remove stuff. However https://github.com/contiki-os/contiki/blob/master/cpu/6502/6502def.h already turns off about everything by default. Maybe IPv6 has some more config options?